### ISSUE TITLE
Customize Horizontal Display

### ISSUE THREAD

**Issue by rjdse** [Darakah/obsidian-timelines#39]
First of all, thanks for the plugin. The concept is awesome. I am using it to help my son create a historical timeline for his Indian history lessons. See below a screenshot for the same.

[IndianHistory](https://github.com/rjdse/TestFiles/blob/main/TL.jpg)

I wanted to ask, whether it is possible to drop a line from the start of each event, to the Timeline Axis. This would make easier to see the chronology since, for events with smaller time range, sometimes it's difficult to judge. Please see below for what I mean should be the desired view -- the illustration meant to convey just the idea, and not exactly meant to be replicated.

[Chronology](https://github.com/rjdse/TestFiles/blob/main/TL_desired.jpg)

Can I seek your guidance in case this can be achieved using the css customisation. If not, if this can be achieved in upcoming versions ...

**Comment by Darakah** [comment]
Hello,
This plugin is using "timeline - vis.js" (https://visjs.github.io/vis-timeline/docs/timeline/).
As such the possible behaviors are the ones supported by vis.js. 
if you specify the **data-type** as "box", this can have a behavior close to what you desire (it projects a vertical line to the timeline axis). i.e. **`data-type='box'`**

I hope this helps.

**Comment by rjdse** [comment]
> Hello, This plugin is using "timeline - vis.js" (https://visjs.github.io/vis-timeline/docs/timeline/). As such the possible behaviors are the ones supported by vis.js. if you specify the **data-type** as "box", this can have a behavior close to what you desire (it projects a vertical line to the timeline axis). i.e. **`data-type='box'`**
> 
> I hope this helps.

Thanks for the response Darakh. Yes this helped a bit, though I wanted the line to drop down from the beginning of the box. But I think this also will work for me :-)

**Comment by rjdse** [comment]
> 

Oh ! Just noticed that when I use the data-type as "box", the time range of the event is not honoured in the created box -- the length of the box does not correspond to the time range of the event -- rather it's just long enough to accomodate the name of the event !! 

So this will not do :-(

**Comment by Darakah** [comment]
Yes, unfortunately that is the behavior of vis.js to my knowledge.
I will try to see if I can add it in the next version.

**Comment by rjdse** [comment]
Very thanks for the clarification Darakh ..

**Comment by rjdse** [comment]
Can you also let me know if the "alternative path option" as mentioned in issue (https://github.com/Darakah/obsidian-timelines/pull/20) is available for use or not.

I tried specifying the path variable in the span syntax, but it doesn't seem to work at all !!

**Comment by rjdse** [comment]
> 

Can anyone please guide if this "alternative path option" has been implemented or not. I am not being able to use this, as per the syntax given in issue #20 (https://github.com/Darakah/obsidian-timelines/pull/20) ...
